package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "Leaderboard",
+  "version": "1.0.0",
+  "description": "The leaderboard website displays scores submitted by different players. It also allows you to submit your score. All data is preserved thanks to the external Leaderboard API service.",
+  "scripts": {
+    "test": "jest",
+    "jest": "jest --watchAll * .js",
+    "build": "webpack",
+    "start": "webpack serve --open",
+    "watch": "webpack --watch"
```

Comment:
- [ ] Good job on scripts. Let's also add the `dev`  to handle the development mode. In the build script kindly also add the mode which should be `production` Check it out below⬇️ 
```
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack --config webpack.config.js --watch"
  }

```